uvm/servlets/admin/config/network/view/Interfaces.js

```diff
@@ -199,6 +199,7 @@ Ext.define('Ung.config.network.view.Interfaces', {
             sourceConfig: {
                 device: { displayName: 'Device'.t() },
                 macAddress: { displayName: 'MAC Address'.t() },
+                vendor: { displayName: 'Vendor'.t() },
```

Comment:
Let's clarify the displayName as "MAC Vendor"

Reply:
Done